src/views/index.njk

```diff
@@ -1,21 +1,153 @@
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="tile">
+        <header>
+            <h2>diaspora* - Friendica - Redmatrix - Hubzilla</h2>
+            <h3>Discover the projects which are composing the federation</h3>
+        </header>
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Project</th>
+                        <th><a href="/diaspora.html">diaspora*</a></th>
+                        <th><a href="/friendica.html">Friendica</a></th>
+                        <th><a href="/redmatrix.html">Redmatrix</a></th>
+                        <th><a href="/hubzilla.html">Hubzilla</a></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th>First release date</th>
+                        <td>11-2010</td>
+                        <td>04-2011</td>
+                        <td>12-2013</td>
+                        <td>08-2015</td>
+                    </tr>
+                    <tr>
+                        <th>Last release date</th>
+                        <td>01-2016</td>
+                        <td>01-2016</td>
+                        <td>09-2015</td>
+                        <td>02-2016</td>
+                    </tr>
+                    <tr>
+                        <th>Nodes</th>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                    </tr>
+                    <tr>
+                        <th>Users</th>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                    </tr>
+                    <tr>
+                        <th>Official website</th>
+                        <td><a href="https://diasporafoundation.org/">diasporafoundation.org</a></td>
+                        <td><a href="http://friendica.com/">friendica.com</a></td>
+                        <td>-</td>
+                        <td><a href="http://hubzilla.org">hubzilla.org</a></td>
+                    </tr>
+                    <tr>
+                        <th>Source code</th>
+                        <td><a href="https://github.com/diaspora/diaspora">Github</a></td>
+                        <td><a href="https://github.com/friendica/friendica">Github</a></td>
+                        <td><a href="https://github.com/redmatrix/redmatrix">Github</a></td>
+                        <td><a href="https://github.com/redmatrix/hubzilla">Github</a></td>
+                    </tr>
+                    <tr>
+                        <th>Licence</th>
+                        <td>AGPL</td>
+                        <td>AGPL</td>
+                        <td></td>
+                        <td></td>
```

Comment:
GitHub is not clear but homepage claims MIT, which the GitHub LICENSE file looks like anyway.